modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
```

Comment:
Lets rename this heading to - `Configure vSphere CSI Topology`

Reply:
Done.